src/Passable.js

```diff
@@ -27,11 +36,27 @@ class Passable {
         this.res = new ResultObject(name);
 
         tests(this.test);
-
-        return this.res;
+        this.runPendingTests();
     }
 
-    test = (fieldName: string, statement: string, test: TestFn, severity: Severity) => {
+    addPendingTest = (test: PassableTest) => this.pending.push(test);
+    clearPendingTest = (test: PassableTest) => {
+
+        // $FlowFixMe
+        this.pending = this.pending.filter((t: PassableTest): boolean => t !== test);
+        if (this.pending.length === 0) {
+            this.res.runCompletionCallbacks();
+        }
+    };
```

Comment:
I don't like the `pending` term here. why not just addTest, clearTest

Reply:
Because addTest/clearTest is lacking context. Where do you add the test to or clear it from?